pom.xml

```diff
@@ -23,12 +23,12 @@
     <properties>
         <cpp.repo.name>metrics</cpp.repo.name>
 
-        <common-bom.version>1.26.0</common-bom.version>
-        <framework-api.version>2.0.3</framework-api.version>
-        <microservice-framework.version>4.1.2</microservice-framework.version>
-
-        <test-utils.version>1.17.2</test-utils.version>
-        <utilities.version>1.13.1</utilities.version>
+        <maven-framework-parent-pom>1.12.3</maven-framework-parent-pom>
```

Comment:
Shouldn't the maven-framework-parent-pom have been set in the <parent> tag and not here?